.github/actions/reusable-python-setup_conda/action.yml

```diff
@@ -15,6 +23,10 @@ runs:
       run: |
         os_conda=$( [[ $RUNNER_OS == "Linux" ]] && echo "linux" || echo "win")
         echo "os_conda=$os_conda" >> $GITHUB_ENV
+    - name: Credentials for Artifactory
+      shell: bash
+      run: |
+        echo "machine ${{ inputs.JFROG_ARTIFACTORY_URL }} login github password ${{ inputs.JFROG_ARTIFACTORY_TOKEN }}" > ~/.netrc
```

Comment:
alternatively, could use `micromamba auth login github -t <token>` (or `-p <token>` ?)
However that requires to first init shells with micromamba, and then call micromamba as a command line from another step...
Well, now that I am writing about ti, does not seem so terrible